Mathlib/Analysis/InnerProductSpace/Spectrum.lean

```diff
@@ -250,7 +250,6 @@ end Version2
 end IsSymmetric
 
 end LinearMap
-
 section Nonneg
```

Comment:
Let's not introduce random changes to the diff.

Reply:
Can I get rid of this?

Reply:
Put the blank line back in.